src/main/java/net/sf/jabref/logic/exporter/MetaDataSerializer.java

```diff
@@ -0,0 +1,57 @@
+package net.sf.jabref.logic.exporter;
+
+import java.util.List;
+import java.util.Map;
+import java.util.TreeMap;
+
+import net.sf.jabref.logic.util.OS;
+import net.sf.jabref.model.MetaData;
+import net.sf.jabref.model.util.ModelStringUtil;
+
+public class MetaDataSerializer {
+
+    /**
+     * Writes all data in the format <key, serialized data>.
+     */
+    public static Map<String, String> getSerializedStringMap(MetaData metaData) {
+    
+        Map<String, String> serializedMetaData = new TreeMap<>();
+    
+        // first write all meta data except groups
+        for (Map.Entry<String, List<String>> metaItem : metaData.getMetaData().entrySet()) {
+    
+            StringBuilder stringBuilder = new StringBuilder();
+            for (String dataItem : metaItem.getValue()) {
+                stringBuilder.append(ModelStringUtil.quote(dataItem, ";", '\\')).append(";");
```

Comment:
I *think* `";"` is (all) the characters that should be escaped (quoted) and `\` is the character to escape with (which will also be escaped). Constant makes sense.